etc/ci/taskcluster-test.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
```

Comment:
I'm not sure if taskcluster already does folding or something like Travis, but it would be nice to add `set -x` or something so that it's easier to figure out which step fails in the case of an error.